ui/package.json

```diff
@@ -38,16 +28,26 @@
     "@fortawesome/pro-regular-svg-icons": "^5.15.3",
     "@fortawesome/pro-solid-svg-icons": "^5.15.3",
     "@fortawesome/react-fontawesome": "^0.1.16",
+    "@testing-library/jest-dom": "^5.16.1",
```

Comment:
Unrelated: it bothers me slightly that these are included in `dependencies` rather than `devDependencies`. I vaguely  recall trying to move them and hitting some kind of issue.. I guess the official line is that the distinction is not very important: https://stackoverflow.com/questions/44868453/create-react-app-install-devdepencies-in-dependencies-section/44872787#44872787